shipstation/api.py

```diff
@@ -110,6 +110,11 @@ def __init__(self, units=None, value=None):
         self.units = units
         self.value = value
 
+        if not value or not isinstance(value, (int, float, Decimal)):
```

Comment:
I know this seems nitpicky, but I strongly prefer that all conditional blocks end with whitespace to show relation (or lack thereof).

```python
if some_condition:
    do_something()

if some_other condition:
    do_something_else()
else:
    fallback()

if a_third_condition:
    do_something()
elif alternative_to_third_condition:
    do_something_else()
else:
    fallback()

```